policyengine_uk/tests/policy/integration/income.yaml

```diff
@@ -0,0 +1,34 @@
+- name: Earned income is taxed correctly
+  period: 2020
+  absolute_error_margin: 0
+  input:
+    employment_income: 100000
+  output:
+    allowances: 12500
+    earned_taxable_income: 87500
+    basic_rate_earned_income_tax: 7500
+    higher_rate_earned_income_tax: 20000
+    add_rate_earned_income_tax: 0
+    earned_income_tax: 27500
+    tax_band: "HIGHER"
+    taxed_income: 87500
+- name: Savings slice of basic rate band
```

Comment:
```suggestion
    taxed_income: 87500

- name: Savings slice of basic rate band
```

Reply:
minor but so much easier to review with spaces in between

Reply:
I will gradually add these into test files as I encounter them, but won't do so proactively, considering the volume of files present in the repo.

Reply:
I would only do it in the test files that are adjusted in the PR -> e.g. I made the adjustments for the universal credit as I went through it. Not a high priority but good to clean up while we are in here